test/test_example.sh

```diff
@@ -19,4 +19,10 @@ export DATABASE_IMAGE_NAME="$MYSQL_IMAGE_NAME"
 ## Use in combination with example compose files 
 export DEFAULT_SUITE_CONFIG="-f ../example/docker-compose.yml -f ../example/docker-compose.extra.yml -f docker-compose.example.yml"
 
+# setup log directory, create "last-ran" file and setup log
+rm -Rf "log/$SUITE"
+mkdir -p "log/$SUITE"
```

Comment:
```suggestion
mkdir -p "log/$SUITE/wikibase"
mkdir -p "log/$SUITE/client"
```

should fix the permission denied thingy

Reply:
This I think does fix it on the Thinkpads, but running as the current user breaks things on Github Actions CI. I've made additional changes in a subsequent branch which moves the screenshot and result file manage directly into the wikibase-selenium-test container such that the same user (root) that creates the files also deletes them, which logically is a better bet. I have not yet done the same with the MediaWiki/Wikibase containers for those logs, but those for whatever reason don't seem to be causing any issue. The other logs are created and maintained by the shell scripts (`log/SUITE/setup.log` and `log/SUITE/SUITE.log`). 

I've created a follow-up PR for this additional changes here: https://github.com/wmde/wikibase-release-pipeline/pull/445